frontend/src/Component/History/SensorHistory.jsx

```diff
@@ -65,6 +78,17 @@ export const SensorHistory = () => {
     return null;
   };
 
+  const ErrorMessage = () => {
```

Comment:
Same as previous comment, This could be turned into something we are able to re-use across the whole site as it's own component.